resources/stylesheets/download.css

```diff
@@ -0,0 +1,51 @@
+.button {
+    display: inline-block;
+    padding: 15px 30px;
+    margin: 10px;
+    border-radius: 5px;
+    color: white;
+    text-decoration: none;
+    font-size: 16px;
+    transition: background-color 0.3s;
+    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+}
+
+.windows {
+    background-color: #0078D7;
+}
+
+.linux {
+    background-color: #FCC624;
+    color: #333;
+}
+
+.mac {
+    background-color: #A4C8E1;
+}
+
+.android {
+    background-color: #A4C639;
+}
```

Comment:
These colour choices are too bright, and have poor contrast with the white text, making for a poor visual experience when using the website.

Coincidentally, I think my dark mode extension which I left on by accident while testing this page has a good selection of colours, so try to go for something more similar to this:

<img src=https://github.com/user-attachments/assets/a5d7a49a-becb-473c-9b54-29dc7a940152 width=300px/>

Additionally, the choices of colour for Mac and Linux make little sense. Perhaps try black and white for Linux and MacOS respectively? My logic is following Tux and the Apple logo's conventional colour palettes.

Reply:
We really can do better than that

Reply:
> We really can do better than that

What are you referring to?

Reply:
css deisgn

Reply:
Clarification:

Having this many big buttons isn't really ideal

We could either detect the platform the wen browser is on, make that the main button, and add smaller elements for the rest of the platforms or just combine the UI elements in a way that looks better if we don't wanna go through tha trouble of adding the OS detective code

Reply:
I was going to get to that, but I wanted to get this out of the way first to avoid conflicting change requests

Reply:
So.... maybe making a design and putting an icon for each system would be a viable option, the point is that I put the color separation to speed up loading 
Also, @kleidis had a great idea about detecting the system in js and it's not difficult, but the size of the button depends on the fact that Windows has two options and so does Android.
Would you show both and put one by default? apk and msvc?

Reply:
Idk what OpenSauce likes for Wibdows default to he 
For now, put MSVC

Android should definitely be the apk

Reply:
If this kind of download functionality is to be added to the website, all relevant downloads must be available.

Reply:
> If this kind of download functionality is to be added to the website, all relevant downloads must be available

You mean that all buttons should be visible no matter what platform?

Reply:
a made changes on windows for now look this

is good?

![msedge_NfjfNVPv3v](https://github.com/user-attachments/assets/e17bddb5-fd6e-46f7-ae75-f8d57df89576)

I don't know about css of buttons yet

Reply:
I'm just throwing this as a snippet

Could we just do it like this?

![Screenshot_20240924_214624_Kiwi Browser.jpg](https://github.com/user-attachments/assets/efc2e4dc-066d-4a73-a946-a7e967590bb8)

Reply:
how about types? we have 2 buttons for windows and android, put side by side?

Reply:
here another way I think before

![image](https://github.com/user-attachments/assets/9283da1e-bf54-4560-894e-6680c7d95373)

Reply:
![image](https://github.com/user-attachments/assets/72cd471d-aac4-4d9b-aedc-f406500e97ed)
Buttons like this?

Reply:
Waitibg for @OpebSauce04 response

Reply:
The second of those options looks good. Could do with a little extra styling to make it feel more integrated with the website but that's something I can handle.

Why are the labels still formatted like filenames by the way? That seems like a pretty strange decision. Just have `Windows (MSVC)`, `Linux`, etc.

Also, the AAB doesn't need to be on the website. It doesn't really have a purpose to the end user.

Reply:
> how about types? we have 2 buttons for windows and android, put side by side?

i think a dropdown might be the better option? maybe